cmd/vulcan-prowler/main.go

```diff
@@ -44,26 +45,47 @@ var (
 	CISCompliance = report.Vulnerability{
 		Summary: "Compliance With CIS AWS Foundations Benchmark (BETA)",
 		Description: `<p>
+			The check did not received the security classification level
+			of the AWS account so the benchmark has been executed against the
+			CIS Level 2.
 			The CIS AWS Foundations Benchmark provides prescriptive
 			guidance for configuring security options for a subset of Amazon Web
 			Services with an emphasis on foundational, testable, and architecture
-			agnostic settings. The services included in the scope are:
+			agnostic settings. The services included in the scope are: IAM, Confing, 
+			CloudTrail, CloudWatch, SNS, S3 and VPC (Default).
 		</p>
 		<p>
-		<ul>
-			<li>IAM</li>
-			<li>Config</li>
-			<li>CloudTrail</li>
-			<li>CloudWatch</li>
-			<li>SNS</li>
-			<li>S3</li>
-			<li>VPC (Default)</li>
-		</ul>
+			The provided recommendations are provided as a result of controls included in
+			the CIS Level 2.
 		</p>
 		<p>
-			The provided recommendations are classified in 2 different levels: 1
-			and 2, being the level 2 intended for environments or use cases where
-			security is paramount.
+			Check the Details and Resources sections to know the compliance status
+			and more details.
+		</p>`,
+		References: []string{
+			"https://d0.awsstatic.com/whitepapers/compliance/AWS_CIS_Foundations_Benchmark.pdf",
+			"https://github.com/toniblyx/prowler",
+			"https://www.cisecurity.org/benchmark/amazon_web_services/",
+		},
+	}
+
+	CISLevel1Compliance = report.Vulnerability{
+		Summary: "Compliance With CIS Level 1 AWS Foundations Benchmark (BETA)",
+		Description: `<p>
+			This account has been checked for compliance with the CIS Level 1 according
+			to its security classification. You can check the security classification of the account in 
+			the details section.
+			</p>
+			<p>
+		    The CIS AWS Foundations Benchmark provides prescriptive
+			guidance for configuring security options for a subset of Amazon Web
+			Services with an emphasis on foundational, testable, and architecture
+			agnostic settings. The services included in the scope are: IAM, Confing, 
+			CloudTrail, CloudWatch, SNS, S3 and VPC (Default).
+		</p>
+		<p>
+			The provided recommendations are provided as a result controls included in
```

Comment:
I would avoid saying provided twice in the same sentence. I also think this sentence is missing an "of".
Maybe we can leave it like: `Recommendations are provided in order to comply with all the controls required by the CIS Level 1.`

Reply:
Makes absolutely sense.